src/homepageExperience/containers/HomepagePythonWizard.tsx

```diff
@@ -8,6 +8,8 @@ import {Overview} from 'src/homepageExperience/components/steps/Overview'
 import {Navigation} from 'src/homepageExperience/components/Navigation'
 
 import {HOMEPAGE_NAVIGATION_STEPS} from 'src/homepageExperience/utils'
+import CreateToken from 'src/homepageExperience/components/steps/CreateToken'
+import InitalizeClient from 'src/homepageExperience/components/steps/InitalizeClient'
```

Comment:
let's keep these exports consistent and export named functions rather than defaults